src/vine_i18n_messages_provider.ts

```diff
@@ -64,7 +64,13 @@ export class I18nMessagesProvider implements MessagesProviderContact {
         ...meta,
       })
     }
-
+    /**
+     * Checking if vine's meta comes with otherField
+     */
+    if (meta?.otherField) {
```

Comment:
The check has to be moved up, next to the `fieldName` translation resolution. Also, you should wrap `meta.otherField` assignment into the conditional.

```ts
if (meta?.otherField) {
      const translatedOtherFieldName = this.#i18n.resolveIdentifier(
        `${this.#fieldsPrefix}.${meta.otherField}`
      )
      if (translatedOtherFieldName) {
        meta.otherField = this.#i18n.formatRawMessage(translatedOtherFieldName.message)
      }
    }
```